tests/virt/cluster/longevity_tests/test_multi_vm_upgrade_and_reboot.py

```diff
@@ -9,6 +9,7 @@
 ]
 
 
+@pytest.mark.gating
```

Comment:
Windows tests are not supposed to run on PSI. Do we still want this?

Reply:
@SiboWang1997   this test is very heavy for gating, it even has a mark:  `pytest.mark.longevity`

Reply:
+1 it doesn't make any sense to have longevity tests in gating...